### PR TITLE
Add chardet

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/chardet/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (synapticarbors) [inline recipes/chardet/meta.yaml]: add:

```
build:
...
   entry_points: 
      - chardetect = chardet.cli.chardetect:main
```

Comment (synapticarbors) [inline recipes/chardet/meta.yaml]: I just realized that the github repo is way ahead of the release, so the `setup.py` isn't reflecting what's in the tarball. It should be `- chardetect = chardet.chardetect:main`

Comment (jschueller) [inline recipes/chardet/meta.yaml]: @synapticarbors ok

**Comment by jakirkham** [comment]
Thanks @jschueller.

**Comment by jakirkham** [comment]
Just saw your [note]( https://github.com/conda-forge/staged-recipes/issues/139#issuecomment-216607826 ) in issue ( https://github.com/conda-forge/staged-recipes/issues/139 ) as it pertains to `chardet`, @dan-blanchard, would you want to be a maintainer of `chardet`? If so, we have a feedstock for it now.

**Comment by dan-blanchard** [comment]
Sure thing. I've been working on a new chardet release on and off for like a year, so it would be good to be a maintainer of the conda recipe too.

**Comment by jakirkham** [comment]
Thanks for helping out. Proposed adding you in PR ( https://github.com/conda-forge/chardet-feedstock/pull/1 ).

**Comment by dan-blanchard** [comment]
Also, @jakirkham, as @sigmavirus24 has previously said, if it's on his list of projects, you can add him as a maintainer, so please add him for chardet too.

**Comment by jakirkham** [comment]
Thanks for the reminder. Done in PR ( https://github.com/conda-forge/chardet-feedstock/pull/2 ).

**Pull request merged**